test/bigml/test/api/utils.clj

```diff
@@ -0,0 +1,56 @@
+;; Copyright 2016 BigML
+;; Licensed under the Apache License, Version 2.0
+;; http://www.apache.org/licenses/LICENSE-2.0
+
+(ns bigml.test.api.utils
+  "Contains a few facilities especially meant tor testing"
+  (:require (bigml.api [core :as api])))
+
+(defn authenticated-connection
+  "Returns a properly authenticated connection.
+  Credentials shall be provided through the BIGML_USERNAME
+  and BIGML_API_KEY environment variables"
+  []
+  (let [username (System/getenv "BIGML_USERNAME")
+        api-key (System/getenv "BIGML_API_KEY")]
+    (api/make-connection username api-key)))
+
+(defn- do-verb
```

Comment:
Thanks a lot for all suggestions, @ashenfad. As to this one (using a map instead of ns-resolve), I clearly see that the solution you suggest has its merits, but it would force me to touch this file and add a new binding symbol/verb/function for each new resource that is added. Even more (sort of) boilerplate would be required in case a new verb were added (this is just hypothetical, at the moment). Do you think that `ns-resolve` is a complete no-no, even in such context where the possibility that it fails resolving a symbol is not bad in itself (since we are unit testing under a controlled number of cases)? Thanks...